package.json

```diff
@@ -32,7 +32,7 @@
     "prepare": "husky install",
     "preview": "electron-vite dev",
     "release": "electron-builder --publish always",
-    "start": "vite --open --port 8080",
```

Comment:
how is this related to this task @roll ? :sweat_smile:

Reply:
Not related